CHANGELOG.md

```diff
@@ -125,7 +127,7 @@ Bugfixes and corrections.
    - Developer friendly interface
    - Standardization for building new endpoint clients
    - Core client implementation
-   - Support for Python’s logging library
+   - Support for Python�s logging library
```

Comment:
?

Reply:
No changes. Not sure why it shows as a difference. I reverted this line and saved before commit.